features/page_objects/journey/api_result_page.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "json"
+
+class ApiResultPage < SitePrism::Page
+  # This is a wrapper page around JSON API results
+
+  element(:json_data, "pre")
```

Comment:
This page object file is similar to `last_email_api_page.rb` and has the same singular element called `"pre"`.

Consider merging the email one into this one

Reply:
I wouldn't do that, as `last_email_page.rb` implement functions specific to that page :)

Reply:
OK :)